src/server/lib/ophan.ts

```diff
@@ -41,7 +147,56 @@ export const record = (event: OphanEvent, config: OphanConfig = {}) => {
   }
 };
 
+/**
+ * @name sendOphanInteractionEventServer
+ * @description Send an interaction event to Ophan from the server side
+ *
+ * @param interaction The interaction to send to ophan
+ * @param config The ophan configuration for this event
+ */
 export const sendOphanInteractionEventServer = (
   interaction: OphanInteraction,
   config?: OphanConfig,
 ) => record(interaction, config);
+
+/**
+ * @name sendOphanComponentEventFromQueryParamsServer
+ * @description Send a component event to Ophan from the server side using the componentEventParams query parameter passed in from the frontend
+ *
+ * @param componentEventParamsQuery The componentEventParams query parameter from the frontend
+ * @param action The OphanAction to send to ophan
+ * @param value The value of the component event to send to ophan
+ * @param consentUUID The user consent UUID from the consentUUID cookie
+ */
+export const sendOphanComponentEventFromQueryParamsServer = (
```

Comment:
it'd be great to get some unit test coverage in place for this method

Reply:
added in lots of tests!